.github/workflows/main.yml

```diff
@@ -60,12 +59,12 @@ jobs:
 
   build:
     runs-on: ubuntu-18.04
-    container: liuguo09/ubuntu-nuttx:18.04
+    env:
+      DOCKER_BUILDKIT: 1
 
     strategy:
       matrix:
-        boards: [arm-01, arm-02, arm-03, arm-04, arm-05, arm-06, arm-07, arm-08, arm-09, arm-10, arm-11, arm-12, arm-13, arm-14, arm-15, mips-riscv-x86, sim]
-
+        boards: [arm-01, arm-02, arm-03, arm-04, arm-05, arm-06, arm-07, arm-08, arm-09, arm-10, arm-11, arm-12, arm-13, arm-14, arm-15, mips-riscv-x86, sim, xtensa]
```

Comment:
Since xtensa's config is very small, let's merge xtensa into mips-riscv-x86?

Reply:
Updated to use this instead.  Not the CI on this will fail until the PR in testing is merged.